openai_agents/run_research_workflow.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+
+from temporalio.client import Client
+from temporalio.common import WorkflowIDReusePolicy
+
+from openai_agents.workflows.research_bot_workflow import ResearchWorkflow
+
+
+async def main():
+    # Create client connected to server at the given address
+    client = await Client.connect(
+        "localhost:7233",
+    )
+
+    # Execute a workflow
```

Comment:
Should all client connections use `with set_open_ai_agent_temporal_overrides:` like the one above?

Reply:
The overrides are only needed in the worker, not in the `run_` program that invokes the workflows. Removing other instances.